godpi_example/example_app.go

```diff
@@ -89,6 +92,7 @@ func main() {
 		select {
 		case <-signalChannel:
 			fmt.Println("Received interrupt signal")
+			fmt.Println(protoCounts)
```

Comment:
this should be printed here already: https://github.com/mushorg/go-dpi/blob/77d9d9f2a2a4f6e30f4ab8e5ac8403936f5cf674/godpi_example/example_app.go#L63